<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimeIpNewDialog</class>
 <widget class="QDialog" name="AnimeIpNewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>118</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新增动漫</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="hLayout_Name">
     <item>
      <widget class="QLabel" name="label_Name">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Name"/>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_Display">
       <property name="styleSheet">
        <string notr="true">QCheckBox
{
  spacing: 0px;
}
QCheckBox::indicator
{
  width: 20px;
  height: 20px;
}
QCheckBox::indicator:checked:enabled
{
  image:url(:/File/img/box_check_enable.png);
}
QCheckBox::indicator:unchecked:enabled
{
  image:url(:/File/img/box_uncheck_enable.png);
}
QCheckBox::indicator:unchecked:disabled
{
  image:url(:/File/img/box_uncheck_disable.png);
}</string>
       </property>
       <property name="text">
        <string>公开</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_Zhuifan">
       <property name="styleSheet">
        <string notr="true">QCheckBox
{
  spacing: 0px;
}
QCheckBox::indicator
{
  width: 20px;
  height: 20px;
}
QCheckBox::indicator:checked:enabled
{
  image:url(:/File/img/box_check_enable.png);
}
QCheckBox::indicator:unchecked:enabled
{
  image:url(:/File/img/box_uncheck_enable.png);
}
QCheckBox::indicator:unchecked:disabled
{
  image:url(:/File/img/box_uncheck_disable.png);
}</string>
       </property>
       <property name="text">
        <string>追番</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayout_Keywords">
     <item>
      <widget class="QLabel" name="label_Keywords">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>关键词</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Keywords"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="vSpacer_Space">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayout_OkCancel">
     <item>
      <spacer name="hSpacer_OkCancel">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Ok">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
